### ISSUE TITLE
has_parent invalid path returns entire lots of pages.

### ISSUE THREAD

**Issue by hagcin** [ryancramerdesign/ProcessWire#1204]
has_parent=/adsfasd/dsf/ (not a real path). 36 results (demo site)
has_parent=/  36 results
has_parent= 36 results

They all ignore "limit=25" or any other limit completely 
using selector test module in processwire 2.6.0 clean install, only other 3rd party module is croppableImages.

has_parent=/about/ (valid selector)
returns 2 pages (the correct result)

Note: I have tried this on larger sites and will return thousands of pages with no ability to limit.

Version:  2.6.0 but I have noticed it earlier.

![image](https://cloud.githubusercontent.com/assets/6478273/7873538/8195f040-0571-11e5-9c75-ad0d1d085c81.png)

**Comment by ryancramerdesign** [comment]
Thanks, I was able to duplicate the issue with has_parent=/ljkef/lkejf (invalid path). It was basically mapping to a has_parent=0, which includes all viewable pages in the site. This one has been fixed and will appear in this weeks commits. I was not able to duplicate the "limit" issue, as that seems to be working properly for me. I'm guessing that may be an issue with the Selector Test module, as I'm testing directly from the API.
